science/eso-flames-fposs/Portfile

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+
+# Place holder for future Port group for ESO packages
+#PortGroup           eso 1.0
+
+set eso.prefix      ${prefix}/libexec/eso
+
+name                eso-flames-fposs
+
+version             115.0.3
+revision            7
```

Comment:
`revision` should be zero for a brand new port.

Reply:
`revision` is only incremented in the event that you are changing something that changes how it builds or results in a difference in the packaged assets, but the `version` isn't changing.

Reply:
> `revision` should be zero for a brand new port.

This is only a brand new port for us; ESO has been using it internally for some time.

Reply:
Ah ok, understood.